src/app/profile/Profile.types.ts

```diff
@@ -3,6 +3,6 @@ import { PlayerResponse } from 'api/types';
 export type ProfileProps = {
   profileDetails: PlayerResponse | undefined;
   isLoading: boolean;
-  isError: boolean;
+  error: unknown;
```

Comment:
Faktycznie nie wiadomo jaki to jest typ?

Reply:
Unknown jest domyślą wartością, ale da się ją określić w useQuery. Zrobiłem to